src/ods-button.js

```diff
@@ -127,14 +123,13 @@ class OdsButton extends LitElement {
         title="${ifDefined(this.title ? this.title : undefined)}"
         name="${ifDefined(this.name ? this.name : undefined)}"
         part="button--modifier"
-        @click=${() => this.buttonCallback()}
         type="${ifDefined(this.type ? this.type : undefined)}"
         .value="${ifDefined(this.value ? this.value : undefined)}"
       >
 
         ${ifDefined(this.svgIconLeft ? this.getIcon(this.svgIconLeft) : undefined)}
-
-        <slot></slot>
+        <!--Slot still clickabled when button disabled for some reason-->
+        <slot @click=${e => {if(this.disabled) e.stopPropagation()}}></slot>
```

Comment:
'disabled' is not a valid attr of 'slot'

putting the '@click' on the 'slot' element is not going to work.

Reply:
I'm using an arrow function for the click event, so the `this` scope is set to the instance of `ods-button` not the `slot`. So `this.disabled` in the click handler is equal to `ods-button.disabled`, not `slot.disabled`. 

Your assumption of the `this` context being on the `slot` element is usually correct in the instance of passing in a reference to a normal function or a method.

Reply:
Instead of using Javascript, I think we can use CSS.

```css
slot {
  pointer-events: none;
}
```

Seems to work locally.

Reply:
BRUH, THAT'S CRAZY

Reply:
With this method the text is still clickable when the button is enabled?

Reply:
Yes, it is. When you click the text the button picks up the click instead of the slot.